SkEditor/SkEditorSelfAddon.cs

```diff
@@ -0,0 +1,157 @@
+        }
+
+        Registries.Connections.Register(new RegistryKey(this, "skUnityConnection"),
+            new ConnectionData("skUnity", 
+                "Used as a documentation provider and a script host (via skUnity Parser)", 
+                "SkUnityAPIKey",
+                GetIcon("skUnity.svg", true),
+                "https://skunity.com/dashboard/skunity-api"));
+        
+        Registries.Connections.Register(new RegistryKey(this, "SkriptMCConnection"),
+            new ConnectionData("SkriptMC", 
+                "Used as a documentation provider", 
+                "SkriptMCAPIKey",
+                GetIcon("SkriptMC.png", false),
+                "https://skript-mc.fr/developer/"));
+        
+        Registries.Connections.Register(new RegistryKey(this, "CodeSkriptPlConnection"),
+            new ConnectionData("skript.pl", 
+                "Used as a script host", 
+                "CodeSkriptPlApiKey",
+                GetIcon("skriptpl.svg", true),
+                "https://code.skript.pl/api-key"));
+        
+        Registries.Connections.Register(new RegistryKey(this, "PastebinConnection"),
+            new ConnectionData("Pastebin", 
+                "Used as a script host", 
+                "PastebinApiKey",
+                GetIcon("Pastebin.svg", true),
+                "https://pastebin.com/doc_api"));
+
+        #endregion
+
+        #region Registries - Sidebar Panels
+        
+        Registries.SidebarPanels.Register(new RegistryKey(this, "ProjectPanel"), ProjectPanel);
+        Registries.SidebarPanels.Register(new RegistryKey(this, "ParserPanel"), ParserPanel);
+
+        #endregion
+
+        #region Registries
+
+        Registries.WelcomeEntries.Register(new RegistryKey(this, "Test1"),
+            new WelcomeEntryData("Test 1", new RelayCommand(() => SkEditorAPI.Logs.Debug("Test 1 clicked")), GetAddonIcon()));
+
+        #endregion
+        
+        Registries.BottomIcons.Register(new RegistryKey(this, "test1"),
+            new BottomIconData()
+            {
+                Text = "Hello there",
+                Clicked = ((sender, args) =>
+                {
+                    SkEditorAPI.Logs.Debug("Hello there!");
+                    
+                    (args.Icon as BottomIconData).IsEnabled = false;
+                    (args.Icon as BottomIconData).Text = "General Kenobi!";
+                    (args.Icon as BottomIconData).IconSource = GetIcon("skUnity.svg", true);
+                }),
+                IconSource = GetAddonIcon()
+            });
+    }
+
+    public List<MenuItem> GetMenuItems()
+    {
+        return [new MenuItem
+        {
+            Header = "SkEditor Test",
+            Icon = new SymbolIcon()
+            {
+                Symbol = Symbol.Attach
+            },
+            Command = new RelayCommand(() => SkEditorAPI.Addons.DisableAddon(this))
+        }];
+    }
+
+    public Version GetMinimalSkEditorVersion()
+    {
+        return new Version(2, 5, 0);
```

Comment:
What do you mean?

Reply:
`Assembly.GetExecutingAssembly().GetName().Version` instead of hard-coded one, changing it in two places is tiring

Reply:
My point was to provide the lowest version the addon can work on, so basically this will need to be changed when 2.6 releases for instance (if it introduces the addon rework).

Reply:
Oh, okay, sorry, I didn't understand it correctly xd